src/scanner.rs

```diff
@@ -207,9 +207,18 @@ impl CodeInfo {
 		self.addToken(NUMBER);
 	}
 
+	fn skipEscape(&mut self) {
```

Comment:
is this function necessary to be separated? it only seems to be called once

Reply:
Yes it's not necessary here, I add this because in case escape char needs to be parsed in the future. If really don't need, remove it

Reply:
alright